diff --git a/third_party/blink/renderer/platform/media/web_media_player_builder.cc b/third_party/blink/renderer/platform/media/web_media_player_builder.cc
index 404c775e9eef2..5adbf6d5d0c49 100644
--- a/third_party/blink/renderer/platform/media/web_media_player_builder.cc
+++ b/third_party/blink/renderer/platform/media/web_media_player_builder.cc
@@ -97,7 +97,10 @@ std::unique_ptr<WebMediaPlayer> WebMediaPlayerBuilder::Build(
     bool is_background_video_playback_enabled,
     bool is_background_video_track_optimization_supported,
     std::unique_ptr<media::Demuxer> demuxer_override,
-    scoped_refptr<ThreadSafeBrowserInterfaceBrokerProxy> remote_interfaces) {
+    scoped_refptr<ThreadSafeBrowserInterfaceBrokerProxy> remote_interfaces,
+    VideoLeadFrameSink video_lead_frame_sink,
+    SawtunaaAudioTapProcessCB sawtunaa_audio_tap_process,
+    SawtunaaAudioTapControlCB sawtunaa_audio_tap_control) {
   CHECK_EQ(&static_cast<FrameFetchContext*>(fetch_context_.get())->frame(),
            frame);
   auto video_frame_compositor = std::make_unique<VideoFrameCompositor>(
@@ -116,7 +119,9 @@ std::unique_ptr<WebMediaPlayer> WebMediaPlayerBuilder::Build(
       use_surface_layer, is_background_suspend_enabled,
       is_background_video_playback_enabled,
       is_background_video_track_optimization_supported,
-      std::move(demuxer_override), std::move(remote_interfaces));
+      std::move(demuxer_override), std::move(remote_interfaces),
+      std::move(video_lead_frame_sink), std::move(sawtunaa_audio_tap_process),
+      std::move(sawtunaa_audio_tap_control));
 }
 
 }  // namespace blink
